Helper/ConfigHelper.php

```diff
@@ -234,6 +235,21 @@ public function getAutocompleteSelector($storeId = null)
         return $this->configInterface->getValue(self::AUTOCOMPLETE_SELECTOR, ScopeInterface::SCOPE_STORE, $storeId);
     }
 
+    /**
+     * Returns config value
+     *
+     * @param $storeId
+     * @return mixed
+     */
+    public function includeNonVisibleProductsInIndex($storeId = null)
+    {
+        return $this->configInterface->getValue(
```

Comment:
@mrahman3177 I think we should use `\Magento\Framework\App\Config\ScopeConfigInterface::isSetFlag` here instead and declare a `bool` return type not `mixed`. Let's also be sure to get in the habit of using return type declarations (PHP 7) and parameter type declarations (PHP 5).